frontend/include/chpl/resolution/resolution-queries.h

```diff
@@ -381,6 +381,12 @@ resolveGeneratedCallInMethod(Context* context,
                              const PoiScope* inPoiScope,
                              types::QualifiedType implicitReceiver);
 
+// tries to resolve an (unambiguous) init=
+const TypedFnSignature* tryResolveInitEq(Context* context,
+                                         const uast::AstNode* ast,
```

Comment:
what is 'ast' here?

Based on looking at the body of tryResolveInitEq, it is used for two purposes:
 1. computing a `Scope*` to use
 2. anchoring error messages to source code

I renaming it to `astForScopeOrErr` and/or adding a comment about how `ast` will be used here will address my concern.